src/back/models/Entity.js

```diff
@@ -290,6 +290,18 @@ function Entity(attributeValues, options) {
     this.Entity.specification.name + '" (it has to be a valid uuid)'
   );
 
+  if (this.permissions) {
+    expect(this.permissions).to.be.an.instanceOf(Object);
+    for (var property in this.permissions) {
+      expect(this.permissions[property]).to.have.any.keys('write', 'read');
+      this.permissions[property] = {
+        read: Boolean(this.permissions[property].read),
```

Comment:
It would be nice to only include `read: true` or `write: true`. This way, the standard missing permission is `undefined`.